src/herder/test/UpgradesTests.cpp

```diff
@@ -843,6 +862,86 @@ TEST_CASE("config upgrades applied to ledger", "[soroban][upgrades]")
                 CONFIG_SETTING_CONTRACT_MAX_SIZE_BYTES);
         REQUIRE(sorobanConfig.maxContractSizeBytes() == 32768);
     }
+
+    SECTION("modify BucketListSizeWindowSampleSize")
+    {
+        auto upgradeSize = [&](uint32_t size) {
```

Comment:
nit: should this be a separate function like in other sections so we can re-use it later?

Reply:
I don't think so. I doubt we'll need to do this upgrade in other tests as this is pretty obscure and this test should be sufficient. This also isn't just an upgrade, but we also modify the BucketList size values before the actual upgrade. I think this should just be limited to this particular test, as I don't think other upgrade tests will need or expect these dummy values.